frontend/views/containers/payments/PaymentRowReceived.vue

```diff
@@ -5,8 +5,20 @@
       strong {{ withGroupCurrency(payment.amount) }}
       payment-not-received-tooltip(v-if='notReceived' :member='payment.displayName')
 
-    template(slot='cellActions')
+      .c-amount-pill-container
+        i18n.pill.is-neutral.hide-tablet Manual
+
+    template(slot='cellMethod')
+      .c-methods-container
+        i18n.pill.is-neutral Manual
+
+    template(slot='cellDate')
       .cpr-date.has-text-1 {{ humanDate(payment.date) }}
+
+    template(slot='cellRelativeTo')
+      .c-relative-to TODO
```

Comment:
I think for this PR we can implement this without too much difficulty.

The group contract has a very useful getter called `periodStampGivenDate` defined in `group.js`:

```js
    periodStampGivenDate (state, getters) {
      return (recentDate: string | Date) => {
        if (typeof recentDate !== 'string') {
          recentDate = recentDate.toISOString()
        }
        const { distributionDate, distributionPeriodLength } = getters.groupSettings
        return periodStampGivenDate({
          recentDate,
          periodStart: distributionDate,
          periodLength: distributionPeriodLength
        })
      }
    },
```

Since all group contract getters are imported into Vuex getters, you can import that getter in this file via `mapGetters`.

Then you just need to pass it the `payment.date` when calling it, and it should return the payment period this payment was in.